index.bs

```diff
@@ -4160,10 +4170,13 @@ To <dfn>Emit a context created event</dfn> given |session| and |context|:
 
 The [=remote end event trigger=] is
 the <dfn export>WebDriver BiDi navigable created</dfn> steps given
```

Comment:
FYI this export link is not correct

Reply:
WDYM?

Reply:
The link works from here https://html.spec.whatwg.org/#:~:text=The%20following%20terms%20are%20defined%20in%20WebDriver%20BiDi%3A%20%5BWEBDRIVERBIDI%5D